docs/esp8266/general.rst

```diff
@@ -151,25 +151,26 @@ SSL/TLS limitations
 ~~~~~~~~~~~~~~~~~~~
 
 ESP8266 uses `axTLS <http://axtls.sourceforge.net/>`_ library, which is one
-of the smallest TLS libraries with the compatible licensing. However, it
+of the smallest TLS libraries with compatible licensing. However, it
```

Comment:
What? Removing articles which I spend so much effort to stuff in? ;-) I'll defer this to @dpgeorge .